### ISSUE TITLE
PDFUA-245: Tab order for pages with annotations incorrectly set

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/pdfa-test-7#245]
Jira issue originally created by user @PaulRayius:

This example demonstrates a failure condition in which the document's [tab order](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#tab-order) for [annotations](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#annotation), such as form fields, links or notes, is not set to "S," so as to follow the order of the document's Structure.

**Comment by zakkinsey** [comment]
@PaulRayius added a link to #193: This issue relates to #193

**Comment by zakkinsey** [comment]
@PaulRayius added a link to #247: This issue relates to #247

**Comment by zakkinsey** [comment]
@PaulRayius added a link to #247: This issue relates to #247

**Comment by zakkinsey** [comment]
@MarkusErle removed a link to #247: This issue relates to #247

**Comment by zakkinsey** [comment]
@PaulRayius assigned issue to self

**Comment by zakkinsey** [comment]
Comment created by @luciatoda:

Arlington error:

name tree Names object was missing when Kids was also missing for Trailer (as XRefStream)-Root (as Catalog)-StructTreeRoot-IDTree (as name-tree)

**Comment by zakkinsey** [comment]
Comment created by @luciatoda:

Fixed by removing IDTree

**Comment by zakkinsey** [comment]
Comment created by @PaulRayius:

Replaced file with one with embedded fonts.  Note:  I needed to change the font of "Submit" from Helvetica Bold to just Helvetica.  (I also had to do this in #147 but, for some reason, not the pass case example.  I hope this is ok.)

**Comment by zakkinsey** [comment]
Comment created by @PaulRayius:

Note:  I checked with Acrobat and, in addition to the Tab order fail, Acrobat fails this due to "improper use of Lbl" (as a child of P, not in a list).  

CommonLook flags this, too, although CommonLook will be changing this functionality, soon.

**Comment by zakkinsey** [comment]
@zakkinsey unassigned issue from @PaulRayius as part of jira-&gt;github migration
